monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/frontends.dashboard.py

```diff
@@ -0,0 +1,50 @@
+from grafanalib.core import (
+    Dashboard, Graph, Row, Target
+)
+from grr_grafanalib_dashboards.util import add_data_source
+from grr_grafanalib_dashboards.config import PANELS
+
+GRR_COMPONENT = "frontend"
+
+dashboard = Dashboard(
+    title="{}s Dashboard".format(GRR_COMPONENT).title(),
+    rows=[
+        Row(panels=[
+            panel(GRR_COMPONENT) for panel in PANELS
+        ]
+        ),
+        Row(panels=[
+          Graph(
+            title="QPS",
+            targets=[
+                Target(
+                    expr='sum(rate({}_request_count_total[1m]))'.format(GRR_COMPONENT),
+                    legendFormat="Requests",
```

Comment:
I'm not sure it makes sense format GRR_COMPONENT inside the metric name. We have frontend_request_count_total metric, but is there worker_request_count_total or admin_ui_request_count_total?

Reply:
You're right; there aren't. Will get rid of that.